src/Templates/src/templates/maui-mobile/Resources/Styles.xaml

```diff
@@ -0,0 +1,164 @@
+        <!--<Setter Property="TextDecorations" Value="None" />-->
+        <!--<Setter Property="FormattedText" Value="" />-->
+        <!--<Setter Property="LineBreakMode" Value="WordWrap" />-->
+        <!--<Setter Property="LineHeight" Value="1" />-->
+        <!--<Setter Property="MaxLines" Value="3" />-->
+        <!--<Setter Property="Padding" Value="0" />-->
+        <!--<Setter Property="TextType" Value="Text" />-->
+
+        <!-- VIEW INHERITED PROPERTIES -->
+        <!--<Setter Property="VerticalOptions" Value="Center" />-->
+        <!--<Setter Property="HorizontalOptions" Value="Center" />-->
+        <!--<Setter Property="Margin" Value="0" />-->
+
+        <!-- VISUALELEMENT INHERITED PROPERTIES -->
+        <!--<Setter Property="Shadow" Value="" />-->
+        <!--<Setter Property="Width" Value="100" />-->
+        <!--<Setter Property="Height" Value="25" />-->
+        <!--<Setter Property="Clip" Value="" />-->
+        <!--<Setter Property="IsVisible" Value="True" />-->
+        <!--<Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource PrimaryLightColor}, Dark={StaticResource PrimaryDarkColor}}" />-->
+        <!--<Setter Property="Background" Value="" />-->
+        <!--<Setter Property="WidthRequest" Value="100" />-->
+        <!--<Setter Property="HeightRequest" Value="25" />-->
+        <!--<Setter Property="FlowDirection" Value="LeftToRight" />-->
+    </Style>
+
+    <Style TargetType="Page" ApplyToDerivedTypes="True">
+
+        <!-- PAGE SPECIFIC PROPERTIES -->
+        <!--<Setter Property="Padding" Value="5" />-->
+
+        <!-- VISUALELEMENT INHERITED PROPERTIES -->
+        <!--<Setter Property="Shadow" Value="" />-->
+        <!--<Setter Property="Width" Value="100" />-->
+        <!--<Setter Property="Height" Value="25" />-->
+        <!--<Setter Property="Clip" Value="" />-->
+        <!--<Setter Property="IsVisible" Value="True" />-->
+        <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource PrimaryLightColor}, Dark={StaticResource PrimaryDarkColor}}" />
+        <!--<Setter Property="Background" Value="" />-->
+        <!--<Setter Property="WidthRequest" Value="100" />-->
+        <!--<Setter Property="HeightRequest" Value="25" />-->
+        <!--<Setter Property="FlowDirection" Value="LeftToRight" />-->
+    </Style>
+
+    <Style TargetType="ScrollView">
+
+        <!-- SCROLLVIEW SPECIFIC PROPERTIES -->
+        <!--<Setter Property="Orientation" Value="Vertical" />-->
+        <!--<Setter Property="ContentSize" Value="" />-->
+        <!--<Setter Property="HorizontalScrollBarVisibility" Value="False" />-->
+        <!--<Setter Property="VerticalScrollBarVisibility" Value="False" />-->
+
+        <!-- PAGE INHERITED PROPERTIES -->
+        <!--<Setter Property="IsClippedToBounds" Value="False" />-->
+        <!--<Setter Property="CascadeInputTransparent" Value="False" />-->
+        <Setter Property="Padding" Value="0" />
```

Comment:
What's the default here? Is the default inconsistent across platforms?

Reply:
I'm not sure / can't find that information super easily, so I'll leave it out for now